sklearn_numba_dpex/kmeans/drivers.py

```diff
@@ -777,4 +814,10 @@ def _load_transposed_data_to_device(self, X, cluster_centers):
             raise ValueError(
                 f"Expected X_layout to be equal to 'C' or 'F', but got {self.X_layout} ."
             )
-        return X_t, dpctl.tensor.from_numpy(cluster_centers.T, device=self.device)
+        return (
+            X_t,
+            dpctl.tensor.from_numpy(sample_weight, device=self.device)
+            if sample_weight is not None
+            else dpctl.tensor.empty(1, dtype=X.dtype, device=self.device),
+            dpctl.tensor.from_numpy(cluster_centers.T, device=self.device),
+        )
```

Comment:
Does this read better?
```suggestion
        sample_weights = (
            dpctl.tensor.from_numpy(sample_weight, device=self.device)
            if sample_weight is not None
            else dpctl.tensor.empty(1, dtype=X.dtype, device=self.device)
        )
        Y_t = dpctl.tensor.from_numpy(cluster_centers.T, device=self.device)
        return X_t, sample_weights, Y_t
```

Reply:
+1 but with `s/sample_weights/sample_weights/` to be consistent with the existing naming.